### PR TITLE
Implement 'multipleOf' and add is_number check to 'minimum'

### PR DESCRIPTION
**Pull request by astj**
part of #9.

I implemented `multipleOf` attribute (ref: http://json-schema.org/latest/json-schema-validation.html#anchor14).
This attribute is formally `divisibleBy` in draft v3. Once I implemented `divisibleBy` in https://github.com/pokutuna/p5-Valiemon/commit/a16bc6a0a0ec2b9c4991c64e2dd8e99c3e277825, but I reverted in https://github.com/pokutuna/p5-Valiemon/commit/b7fb50e4ad5649bdbaf4db4c1fcb781e0900becd . If necessary I can revert it.

And also, I added `is_number` check to `minimum` attribute. This improves test suite compatibility.

### CODE REVIEW

**Comment by astj** [comment]
Please review when you have time :bow::bow: > @hitode909 @pokutuna

File: t/03-attr-minimum.t [hunk 1]
```diff
@@ -68,4 +68,22 @@ subtest 'minimum with object propertie' => sub {
...
+    });
+    ($res, $err) = $v->validate(4);
+    ok $res;
+    is $err, undef;
+
+    ($res, $err) = $v->validate(2);
+    ok !$res;
+    is $err->position, '/minimum';
+
+    ($res, $err) = $v->validate("mio");
```
Comment (hitode909) [inline t/03-attr-minimum.t]: mio

File: t/lib/SuiteRunner.pm [hunk 2]
```diff
@@ -55,7 +55,7 @@ sub run_case {
                 fail $description;
                 diag "schema: @{[ encode_json($schema) ]}";
                 diag "data: @{[ $coder->encode($data) ]}";
-                diag "expect: @{[ $valid ? 'valid' : 'invlid' ]}";
```
Comment (hitode909) [inline t/lib/SuiteRunner.pm:58]: :innocent:

**Comment by hitode909** [comment]
I think, 03-attr-minimum.t is no longer necessary because these test suites cover it.
- https://github.com/json-schema/JSON-Schema-Test-Suite/blob/develop/tests/draft3/minimum.json
- https://github.com/json-schema/JSON-Schema-Test-Suite/blob/develop/tests/draft4/minimum.json

**Comment by hitode909** [comment]
@astj :mount_fuji:  reviewed :mount_fuji:

**Comment by astj** [comment]
> no longer necessary because these test suites cover it

Hmm, I see.
I'll remove validation tests from `03-attr-minimum.t`!

**Comment by astj** [comment]
Removed! https://github.com/astj/p5-Valiemon/commit/6f02c09657e8748a5acb8b59e2a402d92557681b

**Comment by astj** [comment]
@hitode909 Re-review please! :bow::bow::bow:

**Comment by hitode909** [comment]
@astj LGTM! :shipit:

**Comment by astj** [comment]
@hitode909 Thanks! I'll merge!

**Pull request merged**